README.md

```diff
@@ -232,56 +232,56 @@ using the same statement object.
 
 
 ```java
-import java.sql.Connection;
```

Comment:
It looks like Connection is still used below. Is the import no longer needed?

Reply:
Yes, error in copy/past missing that line.  Thanks.